### DISCUSSION TITLE
Conversion to ttl

### DISCUSSION THREAD

**Discussion by LinaNachabe** [cimug-org/CIMTool#68] [General]
I am trying to generate .ttl file of CIM profile in Eclipse using CIMTool. 
Only classes are created? If the profile has a property like numberOfValves:integer, why it doesn't appear in the generated .ttl file?

**Comment by ucaiug-admin** [comment]
@LinaNachabe :   When you state the you were trying to generate a .ttl file of the CIM profile in Eclipse using CIMTool.  Are you using the build in .ttl builder (as per example below)?   

![image](https://github.com/cimug-org/CIMTool/assets/63370413/fba9fd42-bafc-4463-859f-98e07ec418bd)
